courses/fundamentals_of_ada/060_record_types.rst

```diff
@@ -426,6 +444,119 @@ Aggregates with `others`
 Quiz
 ------
 
+What is the result of running this code?
+
+.. code:: Ada
+
+   procedure Main is
+      type Record_T is record
+         A, B, C : Integer := 0;
+      end record;
+
+      V : Record_T := (A => 1);
+   begin
+      Put_Line (Integer'Image (V.A));
+   end Main;
+
+A. ``0``
+B. ``1``
+C. :answer:`Compilation error`
+D. Runtime error
+
+.. container:: animate
+
+   The aggregate should be written as :ada:`(A => 1, others => <>)`
+
+------
+Quiz
+------
+
+What is the result of running this code?
+
+.. code:: Ada
+
+   procedure Main is
+      type My_Integer is new Integer;
+      type Record_T is record
+         A, B, C : Integer := 0;
+         D : My_Integer := 0;
+      end record;
+
+      V : Record_T := (others => 1);
+   begin
+      Put_Line (Integer'Image (V.A));
+   end Main;
+
+A. ``0``
+B. ``1``
+C. :answer:`Compilation error`
+D. Runtime error
+
+.. container:: animate
+
+   All components associated to a value using :ada:`others` must be of the same :ada:`type`.
+
+------
+Quiz
+------
+
+What is the result of running this code?
```

Comment:
This is not a college exam. Too many quizzes in a row breaks up the flow of the class.
I don't think there should be more than 2 quizzes per subsection (if the subsection needs more, then break it into multiple subsections)

Reply:
I had very good feedback on having quizzes and it is always possible to skip them when we're lacking time. I think it's to each his own on that, same as for the labs...

Reply:
As for subsections, sure, I try to break those whenever possible, but it's not always easy to do, especially since most interesting quizzes relate to several concepts at once.